src/Socket.php

```diff
@@ -36,16 +36,12 @@ public function __construct(LoopInterface $loop, $socket, Buffer $buffer = null)
 
     public function getLocalAddress()
     {
-        if ($this->socket !== false) {
-            return stream_socket_get_name($this->socket, false);
-        }
+        return $this->sanitizeAddress(@stream_socket_get_name($this->socket, false));
```

Comment:
For the reference, here's the promised follow-up PR: https://github.com/reactphp/socket/pull/63 :+1: